readme.md

```diff
@@ -133,6 +133,21 @@ queryString.stringify({foo: [1,2,3]});
 // => foo=1&foo=2&foo=3
 ```
 
+#### compareFunction
+
+Type: `function`<br>
+Default: `'none'`
```

Comment:
If the default is `undefined` you can remove this line.

Reply:
ok